buildscripts/cstor-repl-ctrl/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#
+# This Dockerfile builds a recent maya nodebot using the latest binary from
```

Comment:
Please fix this comment. it is referring to nodebot

Reply:
done